### DISCUSSION TITLE
Captain tutorial won't checkmark when commanding someone to the bottom turret

### DISCUSSION THREAD

**Discussion by Starnomaly** [FakeFishGames/Barotrauma#13617] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

So I was at the part in the captain tutorial where I'm supposed to give out orders. It was working perfectly fine up until the moment it wants me to send someone to the bottom turret. When I do that the game doesn't check the box for it so I can't move on with the tutorial. While I did have mods installed I only have one that affects gameplay (Come On In) which should only affect the game during normal gameplay. The rest are simple sound mods. Not sure if this is a one time issue or a reoccurring thing either

### Reproduction steps

1. Start roles tutorial
2. Select captain role
3. Go through tutorial up until point where you command someone to bottom turret
4. If reoccurring bug then the box won't checkmark

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.1.18.1 (Treacherous Tides)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I unfortunately could not find a way to reproduce this. We did however recently run into a bug where the bots in the submarine sometimes wandered into the outpost. Sometimes that resulted in the appropriate bot was outside radio range, which caused the order to be given to an incorrect bot, preventing the objective from being completed. I wonder if that could've been what happened here? We've got a fix coming up for that in the next patch.

**Comment by uaxpasha** [comment]
Same issue. Is coilgun a turret? Then I have the same issue. I might have clicked it before command for it appeared (I clicked randomly at some stuff before orders). The only mod I used is for ukrainian localization.

**Comment by kolhammer** [comment]
Same issue for me. I've just run into this issue without any mods and doesn't matter what I try, I cannot get it to complete the task...

![image](https://github.com/FakeFishGames/Barotrauma/assets/1278724/34884b43-67f4-4b12-8559-a85641214cc0)

**Comment by giists** [comment]
I was also unable to reproduce this. If someone still encounters this, could you include a screenshot of the crewmember list incase the order is still going to the wrong npc?

**Comment by murray-andrew-r** [comment]
> I was also unable to reproduce this. If someone still encounters this, could you include a screenshot of the crewmember list incase the order is still going to the wrong npc?

![image](https://github.com/FakeFishGames/Barotrauma/assets/1470798/790a55fb-3021-44b0-9c15-bd5dd918b385)

I also think I'm having this issue. When I select the "Security Trainee," it seems like I start operating a turret immediately.

![image](https://github.com/FakeFishGames/Barotrauma/assets/1470798/db6c5b87-0976-4d2e-9b7b-7e903fbb7345)


Additional context:

* No mods
* Windows 10
* Game version v1.4.5.0

**Comment by Radezolid** [comment]
I had the same issue, I made the order to man the gun their first priority and it worked.
